File: src/Node.py


class Node:

    def __init__(self, name : str, node_type : str, 
                longitude : float = 0.0, lattitude : float = 0.0):
        self.name = name
        self.node_type = node_type
        self.longitude = longitude
        self.lattitude = lattitude
        self.adjacent = {}

File: src/Node.js


class Node {

    constructor(name, nodeType, longitude = 0.0, latitude = 0.0) {
        this.name = name;
        this.nodeType = nodeType;
        this.longitude = longitude;
        this.latitude = latitude;
        this.adjacent = new Map();
    }
}

export default Node;